hotel-management/src/components/Message/index.tsx

```diff
@@ -0,0 +1,22 @@
+import './index.css';
+
+type Message = {
+  children: string;
```

Comment:
use different naming for this prop for more clear should be content or something else